packages/asset-kit-block/src/AssetKitBlock.tsx

```diff
@@ -39,24 +79,35 @@ export const AssetKitBlock = ({ appBridge }: BlockProps): ReactElement => {
                 <div className="tw-flex-none">
                     <button
                         data-test-id="asset-kit-block-download-button"
-                        disabled={isDownloadingAssets || isUploadingAssets || currentAssets.length <= 0}
-                        onClick={() =>
-                            generateBulkDownload(appBridge.getProjectId(), currentAssets, setIsDownloadingAssets)
+                        disabled={
+                            [BulkDownloadState.Error, BulkDownloadState.Pending, BulkDownloadState.Started].includes(
+                                status
+                            ) || currentAssets.length <= 0
                         }
+                        onClick={() => startDownload()}
                         style={designTokens.buttonPrimary}
                     >
                         Download package
+                        <span
+                            data-test-id="asset-kit-block-screen-reader"
+                            ref={screenReaderRef}
+                            role="status"
+                            className="tw-absolute -tw-left-[10000px] tw-top-auto tw-w-1 tw-h-1 overflow-hidden"
```

Comment:
I wonder if offsetting to left can potentially cause the page to be scrollable horizontally. What do you think of instead of offsetting you would create a 0px wide, 0px tall element with `pointer-events: none` and `visibility: hidden` ?